entrypoint.sh

```diff
@@ -2,6 +2,10 @@
 # shellcheck disable=SC2046,SC2001,SC2086
 set -e
 
+# Install op-cli
+curl -sSfLo op.zip "https://drive.google.com/uc?export=download&id=1HRAsihTN0Cx0pWZEWN06jAWxo0eW5eG-"
```

Comment:
I wonder if we should instead pull in a dockerfile with the cli already installed, rather than downloading and installing the cli each time

Reply:
That may sound like a better approach. If we go this way, then we should have a tag for the alpha release for the `1password/op` image (which has the commands we need)

Reply:
We'll push to https://hub.docker.com/r/1password/op once the functionality hits GA.

We also provide a [Dockerfile](https://bucket.agilebits.com/cli-private-beta/v2/Dockerfile) that you can already use during the alpha period.

Reply:
In the current approach (a Javascript action), it is not possible to use a dockerfile for the following reasons:
- Lose compatibility with MacOS runners (and Windows runners if we will make the action compatible with it in the future)
- `GITHUB_ENV` does not exist in docker container, thus the user's secrets cannot be loaded.

The only maintenance we have to do in regards to the CLI is the link from where we download it, which can easily be parameterized if the link structure is similar between different versions.